docs/src/introduction.md

```diff
@@ -1,186 +1,15 @@
 # Introduction
 
-Stack is an RPN stack-based language built in Rust. It is designed to be simple and easy to learn, while still being powerful and flexible. Stack is built with the hard decisions done in the evaluator, allowing you to do almost anything in-language. Things such as imports, modules, and even the standard library are almost entirely written in Stack.
+Welcome to Stack!
 
-## Features
+## What is Stack?
 
-Though Stack is built to be simple at the top-level, it does not sacrifice power. Here is a quick introduction to the base features of the language:
+*Stack is a highly dynamic stack-based language built in Rust.*
 
-<!-- TODO: split these into the features sections -->
+The goal of Stack is to be an **embeddable language** similar to that of Lua. Stack inherits many **metaprogramming** aspects from popular lisps such as Clojure, **out-of-the-box**. Stack allows the developer to manipulate the **code as data** at runtime without any overhead. **What you see is what you get**.
```

Comment:
> *'Stack allows the developer to manipulate the **code as data** at runtime without any overhead.'*

It might be clearer to say something like, *'[...] without the need for an extra language on top.'*